server/config/.env

```diff
@@ -16,7 +16,7 @@ export CDN_HOST=${CDN_HOST}
 export CDN_URL_CSS=${CDN_HOST}/stylesheets/services/applications-developer
 export CDN_URL_JS=${CDN_HOST}/javascripts/app/applications-developer
 
-export APPLICATIONS_DEVELOPER_SERVICE_BASE_URL=${APPLICATIONS_DEVELOPER_API_CH_GOV_UK_URL}
+export APPLICATIONS_DEVELOPER_SERVICE_BASE_URL=${APPLICATION_DEVELOPER_SERVICE_PORT}
```

Comment:
Should the value be `APPLICATIONS_DEVELOPER_SERVICE_URL` from chs-configs rather than the port number?